assets/base/i18n/en.json

```diff
@@ -351,9 +351,6 @@
   "mobile.post_pre_header.flagged": "Flagged",
   "mobile.post_pre_header.pinned": "Pinned",
   "mobile.post_pre_header.pinned_flagged": "Pinned and Flagged",
-  "mobile.post_textbox.empty.message": "You are trying to send an empty message.\nPlease make sure you have a message or at least one attached file.",
```

Comment:
why are these removed?

Reply:
It was actually impossible to trigger that modal. We check `this.canSend()` at the start of `handleSendMessage` which does an early return any time that modal would've been displayed.